### PR TITLE
#4870 [Manager] The documention of definition should be shown

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4870 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/File/FileSelectorDialog.js [hunk 1]
```diff
@@ -66,39 +60,38 @@ const FileSelectorDialog = React.forwardRef((props, ref) => {
...
   onConfirm: PropTypes.func,
   onUpload: PropTypes.func,
-  selectedFiles: PropTypes.array,
-  tableTitle: PropTypes.string,
+  tableProps: PropTypes.shape({
+    options: PropTypes.shape({
+      selectedFiles: PropTypes.array,
+    }),
+  }),
+  title: PropTypes.string,
```
Comment (eechih) [inline ohara-manager/client/src/components/File/FileSelectorDialog.js:86]: 順便重構

File: ohara-manager/client/src/components/Workspace/Settings/pages/BrokerNodesPage.js [hunk 2]
```diff
@@ -24,19 +24,29 @@ import * as hooks from 'hooks';
...
   const [isSelectorDialogOpen, setIsSelectorDialogOpen] = useState(false);
 
-  const brokerNodesInWorkspace = workspace?.broker?.nodeNames
-    ? filter(
-        map(workspace.broker.nodeNames, nodeName =>
-          find(nodesInWorkspace, node => node.hostname === nodeName),
-        ),
-      )
-    : nodesInBroker;
+  const documentation = useMemo(() => {
```
Comment (eechih) [inline ohara-manager/client/src/components/Workspace/Settings/pages/BrokerNodesPage.js:34]: Cache 已經計算的結果

**Comment by konekoya** [comment]
@eechih So we're only displaying definition for some pages, right? Like streamJar doesn't display this 🤔

**Comment by eechih** [comment]
> So we're only displaying definition for some pages, right? Like streamJar doesn't display this 🤔

目前只顯示有 Definition 的 Page，streamJar 好像沒有相關的 definition？

**Comment by konekoya** [comment]
> 目前只顯示有 Definition 的 Page，streamJar �好像沒有相關的 definition？

Yep, just take a look. Stream doesn't have a definition :(

**Comment by eechih** [comment]
retry manager-e2e

**Pull request merged**